### PR TITLE
Add e2e Test coverage for @BotKube notifier showconfig

### LINKED ISSUE #349: Add test coverage for '@BotKube notifier showconfig` command
**Is your feature request related to a problem? Please describe.**

Test coverage is missing for notifier commands

**Describe the solution you'd like**

`@BotKube notifier showconfig` displays resource configuration that BotKube monitors and notifies about. See https://www.botkube.io/usage/#view-botkube-configuration

Integration tests for validating BotKube command responses are implemented here: https://github.com/infracloudio/botkube/blob/develop/test/e2e/command/botkube.go#L46

Add test case to validate  `@BotKube notifier showconfig` command in the existing table drive test cases

### PR DESCRIPTION
**Pull request by anoopmsivadas**
##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

Add e2e Test coverage for @botkube notifier showconfig

Fixes #349

### CODE REVIEW

File: test/e2e/command/botkube.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func (c *context) testBotkubeCommand(t *testing.T) {
 				"  - statefulsets\n" +
 				"  - storageclasses\n",
 		},
+		"BotKube notifier showconfig": {
+			command:  "notifier showconfig",
+			expected: "<@U023BECGF> notifier showconfig",
```
Comment (PrasadG193) [inline test/e2e/command/botkube.go:88]: @anoopmsivadas The expected response is not valid here. `@BotKube notifier showconfig` should return the resource config in the response. https://www.botkube.io/usage/#view-botkube-configuration. I am wondering how the test is passing though

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:88]: @PrasadG193 hey what should I do next

Comment (PrasadG193) [reply test/e2e/command/botkube.go:88]: @anoopmsivadas I think BotKube is taking some time to respond. Could you try adding some sleep (around 5/10s) before `lastSeenMsg := c.GetLastSeenSlackMessage()`?

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:88]: @PrasadG193 I tried that already, but it didn't help

Comment (PrasadG193) [reply test/e2e/command/botkube.go:88]: Can you post the output after adding sleep?

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:88]: `assert.Equal(t,"test", m.Attachments)` here I added "test" to see the actual output.
```shell 
            --- FAIL: TestRun/command/Test_BotKube_command/BotKube_notifier_showconfig (11.00s)
                botkube.go:119: 
                        Error Trace:    botkube.go:119
                        Error:          Not equal: 
                                        expected: string("test")
                                        actual  : []slack.Attachment([]slack.Attachment(nil))
                        Test:           TestRun/command/Test_BotKube_command/BotKube_notifier_showconfig
```

and the message string part passes with `expected: "<@U023BECGF> notifier showconfig"`

**Comment by pkosiec** [comment]
Hi @anoopmsivadas, I'm closing this pull request because of an inactivity. If you would like to finalize this, please feel free to reopen it once the test case is complete, and it passes with a proper expected response 🙂 Cheers!

**Pull request closed without merging**